core/src/main/java/com/flipkart/gap/usl/core/helper/RetentionPolicyHelper.java

```diff
@@ -26,7 +26,7 @@
         policy.visit(listRetentionPolicyVisitor, list);
     }
 
-    public static <V extends DimensionCollection.DimensionElement> boolean isTimeLimitExceeded(int days, V element) {
-        return element.getUpdated() + TimeUnit.DAYS.toMillis(days) < System.currentTimeMillis();
+    public static <V extends DimensionCollection.DimensionElement> boolean isTimeLimitExceeded(int minutes, V element) {
+        return element.getUpdated() + TimeUnit.MINUTES.toMillis(minutes) < System.currentTimeMillis();
```

Comment:
lets use milliseconds only this will give us most granular control!

Reply:
I will do this change